### DISCUSSION TITLE
Is it possible to make a combo chart with PhpSpreadsheet ?

### DISCUSSION THREAD

**Discussion by ramonBentonSRM** [PHPOffice/PhpSpreadsheet#2209] [Q&A] [answered]
I am trying to make a combo chart with PhpSpreadsheet, I have tried with the PhpWord package but cannot seems to do it, does any one have any solution for this?

Thank you.

**Comment by MarkBaker** [comment, marked as answer]
Have you looked at some of the examples? [Here's one such chart](https://github.com/PHPOffice/PhpSpreadsheet/blob/master/samples/Chart/33_Chart_create_composite.php)
